boba-docs/docs/account-abstraction/2_bundler-api.md

```diff
@@ -5,7 +5,6 @@
 |            |                                                                                      |
 | ---------- | ------------------------------------------------------------------------------------ |
 | ChainID    | 9728                                                                                 |
-| AA bundler | [https://bundler.testnet.bnb.boba.network](https://bundler.testnet.bnb.boba.network) |
```

Comment:
It's a bit odd that this _only_ seems to include BNB.   @sk-enya or @mmontour1306 do you know how these account abstraction pages relate to Boba ETH?

Reply:
Not sure, might be @mmontour1306 can share more thoughts on bundler side!